CHANGELOG.md

```diff
@@ -5,9 +5,27 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-<!--
 ## [Unreleased]
--->
+
+### Added
+
+- Display all descriptor types in device view.
+- Add backend API for USB capture devices.
+- Add support for iCE40-usbtrace capture device.
+- Update documentation for multiple traffic views.
+- Add transaction-level and packet-level views of capture.
+- Add keyboard shortcuts.
+
+### Fixed
+
+- Add connecting lines to test output.
```

Comment:
This should be in `Added`, I think. We didn't fix anything we just added extra testing.